content/master/getting-started/provider-aws.md

```diff
@@ -37,7 +37,7 @@ kind: Provider
 metadata:
   name: provider-aws-s3
 spec:
-  package: xpkg.upbound.io/upbound/provider-aws-s3:v1
+  package: xpkg.crossplane.io/crossplane-contrib/provider-aws-s3:v1
```

Comment:
will these `v1` tags work on `xpkg.crossplane.io`, or do we need to specify a full version?

e.g. right now on https://github.com/crossplane-contrib/provider-upjet-aws/pkgs/container/provider-aws-s3/versions i just see `v1.20.1` published